internal/helper/helper.go

```diff
@@ -21,3 +21,51 @@ func GetFirstElementOf(parent string, children ...string) (key string) {
 
 	return key
 }
+
+func ReadFloat(input interface{}) float64 {
+	data, ok := input.(float64)
+	if !ok {
+		fmt.Printf("[ERROR]: Unable to covert %T to float64 \n", input)
```

Comment:
enhance ur error messages, try giving more context

Reply:
The error message is only capturing the gist of the function on what is wrong. Please suggest what could make it better

Reply:
Done